01/app.js

```diff
@@ -0,0 +1,23 @@
+const users = ['Adam Kowalski', 'Michał Stasiak', 'Andrzej Wasilewski', 'Maria Nowak', 'Zofia Kowalska' ,'Barbara Nowakowska'];
+const oddUsers =[];
+
+oddIndex(users, oddUsers);
+showOdd(oddUsers);
+
+
+function oddIndex(array, evenArray){
+    for(let i=0; i<array.length; i++){
+        if(i%2===1){
+            evenArray.push(array[i]);
+        }
+    }
+}
```

Comment:
Lepiej, aby funkcja zwracała wynik operacji niż modyfikowała zmienne poza funkcją.
To taka uwaga na przyszłość, będziemy o tym jeszcze wspominać przy React o tzw. pure functions